ush/regrid_gsiSfcIncr_to_tile.sh

```diff
@@ -38,33 +41,32 @@ for vi in $( seq 1 "${LSOIL_INCR}" ); do
     soil_incr_vars=${soil_incr_vars}'"slc'${vi}'_inc"',
 done
 
-cat << EOF > regrid.nml
- &config
-  n_vars=${n_vars},
-  variable_list=${soil_incr_vars}
-  missing_value=0.,
- /
- &input
-  gridtype="gau_inc",
-  ires=${LONB_CASE_IN},
-  jres=${LATB_CASE_IN},
-  fname="enkfgdas.sfci.nc",
-  dir="./",
-  fname_coord="gaussian_scrip.nc",
-  dir_coord="./"
-/
-
- &output
-  gridtype="fv3_rst",
-  ires=${CASE_OUT:1},
-  jres=${CASE_OUT:1},
-  fname="sfci",
-  dir="./",
-  fname_mask="vegetation_type"
-  dir_mask="./"
-  dir_coord="./",
- /
-EOF
+n_tims=1
+time_list="06"
+if [[ ${DO_LAND_IAU} = ".true." ]]; then
+    n_tims=0
+    ifhrs=()
+    ifhrsi=()
+    ifhrsf=()
+    IFS=',' read -ra landifhrs <<< "${LAND_IAU_FHRS}"  
```

Comment:
Can we change the fortran regridding code to directly read IAU_FHRS instead of doing the re-formatting?